### DISCUSSION TITLE
如何导入已修改的按键绑定

### DISCUSSION THREAD

**Discussion by Wan-Xing-Star** [Meloong-Git/PCL#4348] [提问] [answered]
有没有办法可以在安装新版本后将旧版本已经设置好的按键导入到新版本中？
（每一次开新版本的第一件事就是设置按键，解决冲突和看之前的按键真的很废时间）

**Comment by MoYuan-CN** [comment, marked as answer]
- 将你旧版本的 `options.txt` 中有关键位的设置复制到新版本的 `options.txt` 中
- 对于 Mod（Masa 系列）：将配置文件从旧版本中复制到新版本的对应位置

**Reply by WForst-Breeze** [reply]
有关键位的设置可以在打开 options.txt 后，使用 Ctrl+F，搜索 `key.keyboard.` 即可

**Comment by Chiloven945** [comment]
~有一个同步option.txt的功能投票的~
